src/app.ts

```diff
@@ -114,23 +105,35 @@ const samlConfig: SamlConfig = {
 };
 
 const acs: AssertionConsumerServiceT = async user => {
-  /* Should validate the user */
-  // Create token
-  const token = crypto.randomBytes(32).toString("hex");
-
-  // Write token in redis
-  redisClient.set(
-    `session-token:${token}`,
-    JSON.stringify(user),
-    "EX",
-    3600, // expire sec 1 h
-    (_, __) => new Error("Error setting session token")
-  );
-
-  // Add token in query string
-  return ResponsePermanentRedirect({
-    href: `${config.ENDPOINT_SUCCESS}?token=${token}`
-  });
+  return fromEither(SpidUser.decode(user))
+    .mapLeft(errorsToError)
+    .chain(spidUser =>
```

Comment:
@AleDore see [this](https://github.com/pagopa/hub-spid-login-ms/pull/1#discussion_r612309197) comment